.devops/azure-templates/api-tests.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  # Required
+  - name: "ENV_FILE"
+    type: string
+
+
+steps:
+  - script: |
+      yarn global add newman
+    displayName: 'Newman installation'
+  - script: |
+      newman run api-tests/checkout-tests/checkout-for-ecommerce-api.tests.json -e ${{ parameters.ENV_FILE }} --ignore-redirects --reporters cli,junit --reporter-junit-export Results/api-TEST.xml
+    displayName: 'Run api test app - env file: ${{ parameters.ENV_FILE }}'
+    continueOnError: true
```

Comment:
`continueOnError` makes the pipeline succeed even in case of failure (it succeeds with warnings).
If you want to make the pipeline to continue to the next step and report failure use a `condition: succeededOrFailed()` on the next step

Reply:
Good point, updated with a0dcf34, you can see a run [here](https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=87661&view=results) 😁